Cohort1/tbd/client/tbd/src/search/SearchPage.js

```diff
@@ -1,7 +1,14 @@
+import './index.css'; 
+import '.search.css';
```

Comment:
This should probably not have the leading `.`; so it would be `import 'search.css'` instead.